app/javascript/mastodon/features/compose/components/compose_form.js

```diff
@@ -33,6 +33,11 @@ const messages = defineMessages({
 export default @injectIntl
 class ComposeForm extends ImmutablePureComponent {
 
+  constructor(props) {
+    super(props);
+    this.composeForm = React.createRef();
```

Comment:
We have never done this. We just use `ref={this.setRef}` and `setRef = c => { this.foo = c; }`

Reply:
Thanks. I modified it.